src/runner.ts

```diff
@@ -39,6 +37,11 @@ const printRouteCollection = (routeCollection: RouteCollection) =>
   `# \`${routeCollection.url}\`
 ${routeCollection.routes.map(printRoute).join('\n')}`
 
+const printIndexEntry = (routeCollection: RouteCollection) => {
+  const headerID = routeCollection.url.replace(/{|}|\//g, '').toLowerCase()
```

Comment:
I think this would be a good place to use a character set:

```suggestion
  const headerID = routeCollection.url.replace(/[{}/]/g, '').toLowerCase()
```

A character set matches any character in it (and backslash does not need to be escaped in character sets)

Reply:
I used the repeated alternatives bc I though it improved readability (reader doesn't need to remember difference between `[]` and `()` for example, but looking at it now, you still need to understand regex at least somewhat so it doesn't really help.